src/forEach.js

```diff
@@ -0,0 +1,16 @@
+const forEach = ( collection, iteratee ) => {
+  if (iteratee === null){
+    return null
```

Comment:
This if statement stands out to me - we're supposed to be using the identity function in the event that iteratee is not provided... [Lodash#foreach](https://lodash.com/docs#forEach)